environs/cloudspec/cloudspec.go

```diff
@@ -64,6 +69,26 @@ func (cs CloudSpec) Validate() error {
 	return nil
 }
 
+// IsLocalhostCloud returns true if the cloud is an LXD localhost cloud.
+func (cs *CloudSpec) IsLocalhostCloud() (bool, error) {
```

Comment:
This lxd specific method does not belong here - cloudspec is a generic construct that needs to be provider agnostic. The method is only called from inside lxd provider anyway.
When we move this logic to lxd package, the check for cloud type == "lxd" become irrelevant.

Reply:
done